cumulusci/tasks/github/release.py

```diff
@@ -16,6 +16,10 @@ class CreateRelease(BaseGithubTask):
             "description": "The managed package version number.  Ex: 1.2",
             "required": True,
         },
+        "version_id": {
+            "description": "The SubscriberPackageVersionId (04t) associated with this release.",
+            "required": True,
+        },
```

Comment:
Let's make this optional. Maybe someone has a public repository but doesn't want to publish their package ids.